recipes/rtg-tools/meta.yaml

```diff
@@ -28,6 +29,14 @@ test:
     - rtg vcfeval --help 2>&1 | grep "Usage"
 
 about:
-  home: https://github.com/RealTimeGenomics/rtg-tools
-  license: BSD
-  summary: RealTimeGenomics Tools -- Utilities for accurate VCF comparison and manipulation
+  home: "https://github.com/RealTimeGenomics/rtg-tools"
+  license: "BSD-2-Clause"
+  license_family: BSD
+  license_file: "LICENSE.txt"
+  summary: "RealTimeGenomics Tools -- Utilities for accurate VCF comparison and manipulation."
+  dev_url: "https://github.com/RealTimeGenomics/rtg-tools"
+  doc_url: "https://realtimegenomics.github.io/rtg-tools/index.html"
+
+extra:
+  identifiers:
+    - biotools:rtg_core
```

Comment:
Hi @mencian,
Why does this have `rtg_core` identifier rather than `rtg_tools`? 

We have RTG Core available as a separate package (it is a superset of RTG Tools). RTG Core is not yet in bioconda, however we would like to add it now that we've also made it available under BSD license. It would be great if you could assist with that - I'm not sure how to handle it, since both tools and core provide a `rtg` command. I guess one approach is to just supercede rtg-tools with rtg-core?